src/config/child_properties.rs

```diff
@@ -56,6 +59,11 @@ pub struct ChildProperties {
 
 impl Parse for ChildProperties {
     fn parse(toml_object: &Value, object_name: &str) -> Option<ChildProperties> {
+        toml_object.check_unwanted(&["child_name", "child_type", "child_prop", "function", "name",
+                                     "status", "signal", "property", "version", "module_name",
+                                     "member", "concurrency"],
```

Comment:
Seems this check also don't need as it checked in `parse_object`.
maybe `parse_object` need comment that it check `child_*` for this?